101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -7,102 +7,118 @@
       "metadata": {
         "description": "The region where resources are deployed"
       },
+      "allowedValues": [
+        "australiaeast",
+        "australiasoutheast",
+        "brazilsouth",
+        "canadacentral",
+        "canadaeast",
+        "centralindia",
+        "centralus",
+        "eastasia",
+        "eastus2",
+        "eastus",
+        "japaneast",
+        "japanwest",
+        "northcentralus",
+        "northeurope",
+        "southcentralus",
+        "southindia",
+        "southeastasia",
+        "uksouth",
+        "ukwest",
+        "westcentralus",
+        "westeurope",
+        "westindia",
+        "westus2",
+        "westus",
+        "[resourceGroup().location]"
```

Comment:
remove this option from the list and remove the defaultValue below - use the parameter file to set the location for CI

Reply:
@bmoore-msft In your previous review,

https://github.com/Azure/azure-quickstart-templates/pull/5266#discussion_r222336148

you told me to add a default value. Now, you told me to remove the default value. What should I do?

Reply:
1) remove the [resourceGroup().location] value from the list of allowedValues
2) remove the defaultValue property for this parameter

You're restricting the locations that are allowed, the location of the resource group may not be in your list of allowed values.

Reply:
@bmoore-msft Like other EventGrid templates, I removed the `allowedValues` property and keep the `defaultValue` property, which was my initial PR. The reason behind keeping the `defaultValue` property is to pass the Travis CI validation. Without `defaultValue`, the template won't be validated.